src/sparseml/transformers/finetune/runner.py

```diff
@@ -151,8 +151,11 @@ def one_shot(self, stage: Optional[str] = None):
         """
         _LOGGER.info("*** One Shot ***")
 
+        tokenized_dataset = self.get_dataset_split("calibration")
+        if "labels" in tokenized_dataset.column_names:
+            tokenized_dataset = tokenized_dataset.remove_columns("labels")
```

Comment:
what is the issue here? if this is relatively safe to keep, then leave a comment on why this specific column needs to be removed